android-components/components/service/contile/src/main/java/mozilla/components/service/contile/ContileTopSitesProvider.kt

```diff
@@ -124,6 +122,12 @@ class ContileTopSitesProvider(
                     throw IOException(e)
                 }
             } else {
+                // If fetch failed, we should check if the set of top sites is still valid
+                // and use it as fallback.
+                val topSites = getCachedTopSitesIfValid(true)
```

Comment:
nit: when invoking functions that have primitive types as parameters, I like to include the named parameter like `getCachedTopSitesIfValid(shouldUseServerMaxAge = true)`. This increases readability outside the context of the IDE quite a bit IMO. Can be handy in reviews or when otherwise viewing source online. 

This really only applies when using a primitive directly, since there is added context when the primitive is assigned to a named variable. For example, I would not apply this same nit to the call below:
`cacheState.getCacheMaxAge(shouldUseServerMaxAge)`